File: src/AHF-API/DataLayer/Model/QuizPuzzles.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model
{

    public class Quizpuzzles
    {
        public int ID { get; set; }

        public List<String> Answers { get; set; }
        public List<String> Questions { get; set; }
        public String CorrectAnswer { get; set; }
        
    }
}

   

File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/Model/QuizPuzzles.java
package com.ahf.antwerphasfallen.Model;

public class QuizPuzzles  {

    String question;
    String answers;
    String correctAnswer;
    String difficulty;

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public String getAnswers() {
        return answers;
    }

    public void setAnswers(String answers) {
        this.answers = answers;
    }

    public String getCorrectAnswer() {
        return correctAnswer;
    }

    public void setCorrectAnswer(String correct) {
        this.correctAnswer = correct;
    }

    public String getDifficulty(){
        return difficulty;
    }
}
